melatonin/component_model.h

```diff
@@ -384,10 +384,6 @@ namespace melatonin
                     }
                 }
             }
-            else
-            {
-                jassertfalse;
-            }
```

Comment:
were you running into this? was it a separate issue or same?

Reply:
Same issue, component is deleted, so selectedComponent is now nullptr, so property changes can't be applied